src/pages/debrief/components/vehicle-checks-card/vehicle-checks-card.ts

```diff
@@ -7,7 +7,7 @@ import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
 import { getTests } from '../../../../modules/tests/tests.reducer';
 import { getCurrentTest } from '../../../../modules/tests/tests.selector';
 import { getTestData } from '../../../../modules/tests/test-data/test-data.reducer';
-import { getVehicleChecks } from '../../../../modules/tests/test-data/test-data.selector';
+import { getVehicleChecks } from '../../../../modules/tests/test-data/cat-b/test-data.cat-b.selector';
```

Comment:
The vehicle-checks-card is not cat-b specific. Why do we use cat-b selector here?

Reply:
getVehicleChecks relies on CatB Unique. Do you think there might be a better way to do this? Or do we need category specific cards?

Reply:
We will need category specific cards, yes. The vehicle-checks-card for cat-be will be different.